test/built-ins/Temporal/ZonedDateTime/prototype/until/valid-increments.js

```diff
@@ -0,0 +1,81 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.zoneddatetime.prototype.until
+description: Valid increments are based on divisibility.
+features: [Temporal]
+---*/
+
+const earlier = Temporal.ZonedDateTime.from('2019-01-08T09:22:36.123456789+01:00[+01:00]');
+const later = Temporal.ZonedDateTime.from('2021-09-07T13:39:40.987654321+01:00[+01:00]');
```

Comment:
Can use `0n` and `1_000_000_000_000_000_000n` here as well because the times don't matter.

Reply:
Done